azext_iot/tests/central/test_iot_central_unit.py

```diff
@@ -383,6 +387,208 @@ def test_should_return_fileupload(self, mock_fileupload_svc):
         assert fileupload.connection_string == self._fileupload.connection_string
 
 
+class TestCentralQueryProvider:
+    _query_response = QueryReponseV1_1_preview(
+        load_json(FileNames.central_query_response_file)
+    )
+
+    @mock.patch("azext_iot.central.services.query")
+    def test_should_return_query_response(self, mock_query_svc):
+        # setup
+        provider = CentralQueryProvider(
+            cmd=None,
+            query=None,
+            app_id=app_id,
+            api_version=ApiVersion.v1_1_preview.value,
+        )
+        mock_query_svc.query_run.return_value = self._query_response
+
+        # act
+        query_response = provider.query_run()
+        # verify
+        assert mock_query_svc.query_run.call_count == 1
+        assert query_response.results == self._query_response.results
+
+
+class TestCentralDestinationProvider:
+    _destinations = list(load_json(FileNames.central_destination_file))
+
+    @mock.patch("azext_iot.central.services.destination")
+    def test_should_return_destinations(self, mock_destination_svc):
```

Comment:
can you include the method name in the function name? This will help with associating a test with list vs get

Reply:
Include the method name.